Rakefile

```diff
@@ -65,7 +65,12 @@ task :default do
 end
 
 task :spec do
-  sh %{rspec #{ENV['TEST'] || ENV['TESTS'] || 'spec'}}
+  tests = ENV['TEST'] || ENV['TESTS']
+  unless tests
+    modules_tests = Dir.glob("lib/puppet/vendor/modules/*/spec").join(' ')
```

Comment:
Since acceptance tests will live under `spec/acceptance`, this glob should be more specific to be `*/spec/{unit,integration}`